server/api/reviews.js

```
@@ -0,0 +1,37 @@
+const router = require("express").Router();
+const { Review, Package, User } = require("../db/models");
+module.exports = router;
+
+router.get("/", (req, res, next) => {
+  Review.findAll()
+  .then(reviews => res.json(reviews))
+  .catch(next);
+});
+
+router.post("/", (req, res, next) => {
+  const review = Review.create(req.body)
+  const user = User.findById(req.user.id)
+  Promise.all([Review, User])
```

This should throw an error because you reuse the names of the models here